src/resources/Tutorials.js

```diff
@@ -1,17 +1,70 @@
-import React from "react";
+import PropTypes from "prop-types";
+import { useState } from "react";
+import Youtube from "react-youtube";
 import "./resources.css";
 
+const fakeData = require("./fakeData");
+
+// tags is currently unused
+function ToggleItem({ id, title, description, url, tags }) {
+	const [showButton, setShowButton] = useState(false);
+
+	return (
+		<li className="resource-list-item" key={id}>
+			<button
+				type="button"
+				className={`resource-list-title ${
+					showButton && "resource-list-title-active"
```

Comment:
See comment in `resources.css` at line 116. Change `resource-list-title-active` to `active`. If you open dev mode on browser and click on the tutorial title, you'll see `className="resource-list-title resource-list-title-active"`. It's not quite readable.